### ISSUE TITLE
Overall lighting too bright

### ISSUE THREAD

**Issue by daves-ssu** [Piraxus/OrbiterSkyboltClient#9]
The overall lighting in Skybolt seems to be a bit on the bright side of things and needs to be toned down somewhat,

**Comment by matthew-reid** [comment]
Yeah I agree. There needs to be dynamic brightness and white balance adjustment to simulate the human eye, something like [this](https://outerra.blogspot.com/2011/07/white-balance.html).

**Comment by daves-ssu** [comment]
Yes, I believe that in photographic terms it is called Automatic Gain Control or AGC. It is also known as Automatic Exposure Control, where some kind of logic controls the opening of the iris on the camera.
